src/components/Suitcase/TapIcon.svg

```diff
@@ -0,0 +1,11 @@
+</defs>
+</svg>
```

Comment:
oo should this happen :0

Reply:
I think running prettier should resolve? forgot to do that